dev/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+service:
+  extends:
+    file: ../common.yml
+    service: service
+  volumes:
+    - ./payments-service/:/srv/payments-service
+  links:
+    - solitude:solitude
+
+solitude:
+  extends:
+    file: ../common.yml
+    service: service
+  volumes:
+    - ./solitude/:/srv/solitude
+  links:
+    - mysql:mysql
+
+mysql:
```

Comment:
yup although pedantically speaking it increases the line count in all these files :)